playground/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "dev": "vuepress dev",
+    "dev": "NODE_OPTIONS=--openssl-legacy-provider vuepress dev",
```

Comment:
If this is needed it means we are not compatible with 0.18 by default. I would remove the support for node 18

Reply:
Ok, I'll adjust the PR